airflow/contrib/example_dags/example_gcp_video_intelligence.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that demostrated operators for the Google Cloud Video Intelligence service in the Google
+Cloud Platform.
+
+This DAG relies on the following OS environment variables
+
+* GCP_BUCKET_NAME - Google Cloud Storage bucket where the file exists.
+"""
+import os
+
+# [START howto_operator_vision_retry_import]
+from google.api_core.retry import Retry
+
+# [END howto_operator_vision_retry_import]
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.gcp_video_intelligence_operator import (
+    CloudVideoIntelligenceDetectVideoLabelsOperator,
+    CloudVideoIntelligenceDetectVideoExplicitContentOperator,
+    CloudVideoIntelligenceDetectVideoShotsOperator,
+)
+from airflow.operators.bash_operator import BashOperator
+
+default_args = {"start_date": airflow.utils.dates.days_ago(1)}
+
+# [START howto_operator_video_intelligence_os_args]
+GCP_BUCKET_NAME = os.environ.get(
+    "GCP_VIDEO_INTELLIGENCE_BUCKET_NAME", "test-bucket-name"
+)
+# [END howto_operator_video_intelligence_os_args]
+
+
+# [START howto_operator_video_intelligence_other_args]
+INPUT_URI = "gs://{}/video.mp4".format(GCP_BUCKET_NAME)
+# [END howto_operator_video_intelligence_other_args]
+
+
+with models.DAG(
+    "example_gcp_video_intelligence",
+    default_args=default_args,
+    schedule_interval=None,  # Override to match your needs
+) as dag:
+
+    # [START howto_operator_video_intelligence_detect_labels]
+    detect_video_label = CloudVideoIntelligenceDetectVideoLabelsOperator(
+        input_uri=INPUT_URI,
+        output_uri=None,
+        video_context=None,
+        retry=Retry(maximum=10.0),
```

Comment:
```suggestion
```
The retry object for each method has specific default configurations. Now, you overwrite values. It is better not to give anything to use smart default values.